.palantir/revapi.yml

```diff
@@ -109,6 +109,12 @@ acceptedBreaks:
     - code: "java.method.addedToInterface"
       new: "method org.apache.iceberg.ReplacePartitions org.apache.iceberg.ReplacePartitions::validateNoConflictingDeletes()"
       justification: "Accept all changes prior to introducing API compatibility checks"
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.iceberg.UpdateTableStatistics org.apache.iceberg.Table::updateStatistics()"
+      justification: "new API method"
+    - code: "java.method.addedToInterface"
+      new: "method org.apache.iceberg.UpdateTableStatistics org.apache.iceberg.Transaction::updateStatistics()"
+      justification: "new API method"
```

Comment:
I don't think we want to mix in changes to the public API. It should add the necessary structures to `TableMetadata` only.

Reply:
I don't think I understand. I thought this PR was extracted from that one and should contain a subset.

Just let me know which PR you're using for the TableMetadata changes. That's the one we'll review and get in first. We want to do the implementation before the API.

Reply:
I've un-based this PR from the other one.